tests/scripts/remote-scripts/run-tests.sh

```diff
@@ -6,4 +6,6 @@ source ~/vars.env
 
 echo "export PATH=$PATH:/usr/local/go/bin" >> $HOME/.profile && . $HOME/.profile
 
-cd nginx-gateway-fabric/tests && make test TAG=${TAG} PREFIX=${PREFIX} NGINX_PREFIX=${NGINX_PREFIX} NGINX_PLUS_PREFIX=${NGINX_PLUS_PREFIX} PLUS_ENABLED=${PLUS_ENABLED} GINKGO_LABEL=${GINKGO_LABEL} GINKGO_FLAGS=${GINKGO_FLAGS} PULL_POLICY=Always GW_SERVICE_TYPE=LoadBalancer GW_SVC_GKE_INTERNAL=true NGF_VERSION=${NGF_VERSION}
+cd nginx-gateway-fabric && git pull
```

Comment:
Since I'm adding a script to rsync local changes, I don't think we want `git pull` here, since it will just overwrite those changes.

Reply:
That makes sense, I've removed it 👍🏼